docs/02-tutorial.mdx

```diff
@@ -0,0 +1,281 @@
+
+:::
+
+<!-- TODO: add link to API doc and more details -->
+
+### Check Out our Demo Web App
+
+Need to see a more advanced use case? We got you covered.
+
+:::tip Check it out
+
+See our Demo Web App in action on [blindnet-io.github.io](https://blindnet-io.github.io/privacy-components-web/demos/devkit-simple-tutorial/) and read its code in the [privacy-components-web repository](https://github.com/blindnet-io/privacy-components-web/tree/main/demos/devkit-simple-tutorial).
+
+:::
+
+We keep this project as minimalist as possible for the sake of simplicity. It only use [lit](https://lit.dev/), with [Carbon Web Components](https://web-components.carbondesignsystem.com/) and the [Vaadin Router](https://hilla.dev/docs/routing/router). The build system has been set up using the [Open WC generator](https://open-wc.org/guides/developing-components/getting-started/) (refer to the README for more information).
+
+You can either directly clone the whole monorepo locally:
+
+```bash
+git clone git@github.com:blindnet-io/privacy-components-web.git
+cd privacy-components-web/demos/devkit-simple-tutorial/
+```
+
+or use [git sparse-checkout](https://git-scm.com/docs/git-sparse-checkout) to only download the project itself:
+
+```bash
+mkdir devkit-web-demo
+cd devkit-web-demo
+git init
+git remote add -f origin git@github.com:blindnet-io/privacy-components-web.git
+git config core.sparsecheckout true
+echo "demos/devkit-simple-tutorial" >> .git/info/sparse-checkout
+git pull origin main
+```
+
+Then, you'll need to install the project's dependencies using classic yarn (v1) or npm:
+
+```bash
+yarn install
+# OR
+npm install
+```
+
+Then, start the development server:
+
+```bash
+yarn start
+# OR
+npm start
+```
+
+and open the application in your browser using the given URL.
+
+:::info
+
+The next steps have been already developed in this demo project. You can therefore refer to it whenever in doubt.
+
+:::
+
+## 3. Assert Your Privacy Rights
+
+Now, open the demo web app or static page in your browser to see this component in action:
+
+![first screen of the PRCI component](/img/captures/prci-entry.png)
+
+Here, you can see your users can directly choose from a wide range of privacy request types.
+
+This includes the classical consultation (access), deletion and modification of their data, of course, but also more advanced operations, including but not limited to:
+
+- requesting the portability of their data to a compatible system
+- advanced management of their consents (by objecting to and restricting proceeding of their data, or revoking previously given consents)
+- and retrieving more generic privacy-related information regarding your organization (via the "transparency" request type).
+
+:::note
+
+In accordance with GDPR and other legislations, these are only requests. **You** determine how those are handled and answered, manually or automatically.
```

Comment:
```suggestion
These are just requests. **You** determine how those are handled and answered, manually or automatically.
```

Reply:
Addressed in https://github.com/blindnet-io/blindnet.dev/pull/40